packages/components/src/theme/ComposerTheme/Components/Tooltip.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentStyleConfig } from '@chakra-ui/react';
+import { cssVar } from '@chakra-ui/theme-tools';
+
+const $secondaryColor = cssVar('chakra-colors-text-secondary');
+const $arrowBg = cssVar('popper-arrow-bg');
+
+export const Tooltip: ComponentStyleConfig = {
+  baseStyle: ({ theme }) => ({
+    bg: 'text-secondary',
+    [$arrowBg.variable]: $secondaryColor.reference,
```

Comment:
do we have to create a `cssVar` to reference the color?

Reply:
yes: I have tried using the color directly and chakra won't infer the value of the color for CSS variables